crowdin_api/api_resources/projects/resource.py

```diff
@@ -372,3 +375,114 @@ def edit_project_file(
             ),
             request_data=data,
         )
+
+    def get_strings_exporter_path(
+        self,
+        projectId: int,
+        systemStringExporterSettingId: Optional[int] = None
+    ):
+        if systemStringExporterSettingId is None:
+            return f"projects/{projectId}/strings-exporter-settings"
+        return f"projects/{projectId}/strings-exporter-settings/{systemStringExporterSettingId}"
+
+    def list_project_strings_exporter(
+        self,
+        projectId: int,
+        page: Optional[int] = None,
+        offset: Optional[int] = None,
+        limit: Optional[int] = None,
+    ):
+        """
+        List Project Strings Exporter Settings.
+
+        Link to documetation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.strings-exporter-settings.getMany
+        """
+        params = self.get_page_params(page=page, offset=offset, limit=limit)
+        return self._get_entire_data(
+            method="get",
+            path=self.get_strings_exporter_path(projectId=projectId),
+            params=params,
+        )
+
+    def add_project_strings_exporter(
+        self,
+        projectId: int,
+        format: str,
+        settings: Union[
+            AndroidStringsExporterSettings,
+            MacOSXStringsExporterSettings,
+            XliffStringsExporterSettings,
+        ],
+    ):
+        """
+        Add Project Strings Exporter Settings.
+
+        Link to documetation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.strings-exporter-settings.post
+        """
+        return self.requester.request(
+            method="post",
+            path=self.get_strings_exporter_path(projectId=projectId),
+            request_data={"format": format, "settings": settings},
+        )
+
+    def get_project_strings_exporter(
+        self, projectId: int, systemStringsExporterSettingId: int
+    ):
+        """
+        Get Project Strings Exporter Settings
+
+        Link to documetation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.strings-exporter-settings.get
+        """
+        return self.requester.request(
+            method="get",
+            path=self.get_strings_exporter_path(
+                projectId=projectId,
+                systemStringExporterSettingId=systemStringsExporterSettingId,
+            ),
+        )
+
+    def delete_project_strings_exporter(
+        self, projectId: int, systemStringsExporterSettingId: int
+    ):
+        """
+        Delete Project Strings Exporter Settings.
+
+        Link to documetation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.strings-exporter-settings.delete
+        """
+        return self.requester.request(
+            method="delete",
+            path=self.get_strings_exporter_path(
+                projectId=projectId,
+                systemStringExporterSettingId=systemStringsExporterSettingId,
```

Comment:
`systemStringExporterSettingId` -> `systemStringExporterSettingsId`

please fix this everywhere in the code